### DISCUSSION TITLE
Mushroom cards under precise

### DISCUSSION THREAD

**Discussion by gwww** [piitaya/lovelace-mushroom#1583] [General]
I saw the PR https://github.com/piitaya/lovelace-mushroom/pull/1577 to support precise so I tried converting some of my cards to precise. What I had is Mushroom template cards in a grid card as seen below:

<img width="505" alt="Screenshot 2024-11-10 at 17 26 30" src="https://github.com/user-attachments/assets/bf20a7c9-4990-41e9-b1a0-b12e28185b71">

I then created a new section and instead of using the Grid card I just put the cards into the section using precise, as seen below:

<img width="506" alt="Screenshot 2024-11-10 at 17 26 42" src="https://github.com/user-attachments/assets/e1bc87c3-3321-4629-8958-9218f1174b86">

The width of the cards is fine - love the new precise mode for that. The cards are taller though.  Changing the precise height from 2 to 1 row did not work - the text in the cards overflows the card.

My question is, can the height of cards be more precisely controlled? I like the compactness of using the Grid card but I like the flexibility of using a section with individual cards (not nested in a Grid card).

I'm not sure if this is a feature request or a bug. I'm not sure if this is a Mushroom thing or just a general HA thing. Any advice on where to take this next would be helpful. I guess dropping it is also an option - just that I thought precise would give more precision to height and not only width.

Last point. These template cards are on my main mobile dashboard. When their height is taller I have to scroll my dashboard to see cards below. It's nice how the grid card keeps the template cards very compact.

**Comment by S-360** [comment]
Having the same issue since version 4.1.0, completed changed the layout of the dashboard for me. Rolling back to version resolved the issue 4.0.8.

Created a bug report here https://github.com/piitaya/lovelace-mushroom/issues/1586
